engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -97,7 +97,7 @@
     private LocalPlayer localPlayer;
     @In
     private WorldProvider worldProvider;
-    private Camera playerCamera;
+    public Camera playerCamera;
```

Comment:
How about you get it from the WorldRenderer then? It's the object that instantiates the camera in the first place and you can find it on the Context/CoreRegistry. No point in going through intermediaries if it can be avoided.